infra/core/database/azure-cache-for-redis.bicep

```diff
@@ -46,6 +46,16 @@ type PrivateEndpointSettings = {
   subnetId: string
 }
 
+// From: infra/types/RedisUser.bicep
+@description('Type describing the user for redis.')
+type RedisUser = {
+  @description('The object id of the user.')
+  objectId: string
+
+  @description('The alias of the user')
+  alias: string
+}
```

Comment:
You probably want to add accessPolicyName here - that way your core module can assign each user to a different access policy if needed.   Add

accessPolicyName: 'Data Owner' | 'Data Contributor' | 'Data Reader'